### PR TITLE
Products

### PR DESCRIPTION
**Pull request by ashifkhn**
Product page, wishlist and navigation added. 

Latest deploy log | https://app.netlify.com/sites/sweet-macaron-54c627/deploys/624892b1813423000875774b
-- | --
Deploy Preview | https://deploy-preview-5--sweet-macaron-54c627.netlify.app

### CODE REVIEW

**Comment by Pratham82** [comment]
Functionality changes:
1. Add a products link on Shop now button on the homepage.
2. Add functionality to the filters.

UI Changes:
3. Add a border to the product card, make it as the product data looks inside the card
4. Add fixed height and with to the product card
image

Rendering products and Cart mgmt working fine, just add pricing card for showing total cost. And work on overall product styling.

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,24 @@
 import "./App.css";
-import logo from "./logo.png";
+import { BrowserRouter, Routes, Route, } from "react-router-dom";
 import Header from "./components/Header";
-import Homepage from "./pages/home/Homepage";
+import HomePage from "./pages/HomePage/HomePage";
+import { ProductPage } from "./pages/ProductsPage/ProductPage";
+import { CartPage } from "./pages/CartPage/CartPage";
+import { LoginPage } from "./pages/LoginPage/LoginPage";
+import { WishListPAge } from "./pages/WishListPage/WishListPAge";
```
Comment (poojavishnoi) [inline src/App.js:8]: Here, you can import in single line by making index.js in pages and exporting from there.This is called barrel importing.

File: src/StyleClasses/button.css [hunk 2]
```diff
@@ -32,3 +32,27 @@
...
+  /* width: 6rem; */
+  padding: 0.6rem 0.8rem;
+  border: 2px solid black;
+  border-radius: 0.2rem;
+  border: 0.5rem;
+  cursor: pointer;
+}
+
+.btn_secondary2 {
+  background-color: #fff;
```
Comment (poojavishnoi) [inline src/StyleClasses/button.css:49]: use variable for colors and background color

File: src/pages/ProductsPage/productspage.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  box-shadow: 0 0.2rem 0.5rem 0 rgb(0 0 0 / 50%);
+}
+
+.buttons {
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+}
+/* 
+.container {
```
Comment (poojavishnoi) [inline src/pages/ProductsPage/productspage.css:40]: remove commented code

File: src/components/ProductView.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useContext } from "react";
+import "./Style/productview.css";
+import { Cart } from "../context/Context";
+
+export const ProductView = ({ prod }) => {
```
Comment (poojavishnoi) [inline src/components/ProductView.jsx:6]: you can deconstruct prod here only

**Review by poojavishnoi — APPROVED** [review]
Left few suggestions. Other than that everything is good.

**Comment by ashifkhn** [comment]
> Functionality changes:
> 
> 1. Add a products link on Shop now button on the homepage.
> 2. Add functionality to the filters.
> 
> UI Changes: 3. Add a border to the product card, make it as the product data looks inside the card 4. Add fixed height and with to the product card image
> 
> Rendering products and Cart mgmt working fine, just add pricing card for showing total cost. And work on overall product styling.


> Functionality changes:
> 
> 1. Add a products link on Shop now button on the homepage.
> 2. Add functionality to the filters.
> 
> UI Changes: 3. Add a border to the product card, make it as the product data looks inside the card 4. Add fixed height and with to the product card image
> 
> Rendering products and Cart mgmt working fine, just add pricing card for showing total cost. And work on overall product styling.

I'll ab working on it in the next pr

**Comment by ashifkhn** [comment]
> Left few suggestions. Other than that everything is good.

ill be working on your suggestions in next pr

**Pull request merged**